app/helpers/react_components/student/solution_summary.rb

```diff
@@ -34,9 +35,30 @@ def links
           community_solutions: "#",
           learn_more_about_mentoring_article: "#",
           mentoring_info: "#",
-          complete_exercise: Exercism::Routes.complete_api_solution_url(solution.uuid)
+          complete_exercise: Exercism::Routes.complete_api_solution_url(solution.uuid),
+          share_mentoring: "https://some.link/we/need/to-decide-on",
```

Comment:
Could this have the text TODO in it somewhere, to be able to find it later? Or do you expect to update it in this PR?